### PR TITLE
Fix typos and use clearer language in the README

### PR DESCRIPTION
**Pull request by reuvenpo**
The README was very hard to read as the language in it was unclear, and missed many prepositions. I cleaned up the obvious issues, made some stylistic improvements, and made a couple guesses at the original intention of the text. In one paragraph i was not able to understand the intention at all and i left a note in it.

I hope this PR makes the document easier for people to go through so we can collaborate better :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,68 +2,72 @@
...
+## What Is InterWasm DAO?
 
-CosmWasm was born at Hackatom Berlin 2019 by the hands of several code magicians. Initially it was just testing a possibility.
+CosmWasm was born at Hackatom Berlin 2019 by the hands of several code magicians. Initially it was just testing a possibility...
 
-Now CosmWasm spans many blockchains, communities, and continents. As today, the technology powers and secures trillions of dollars. Many more eyes looking into CosmWasm, either as smart contract devs, or blockchains that desires Cosmos and CosmWasm capabilities.
+Now, CosmWasm spans many blockchains, communities, and continents. Today, the technology powers and secures trillions of dollars. Many more eyes are looking into CosmWasm, either as smart contract devs, or blockchains that desire Cosmos and CosmWasm capabilities.
 
-CosmWasm is first citizen cross chain platform, and a value owned and accrued cross chain. 
+CosmWasm is first citizen cross chain platform, and a value owned and accrued cross chain. EDITOR NOTE: I don't understand what this says.
```
Comment (reuvenpo) [inline README.md:25]: This is the line that i was completely unable to understand.

Comment (orkunkl) [reply README.md:25]: Yes you are right. Once merged I will rephrase this one

File: README.md [hunk 2]
```diff
@@ -2,68 +2,72 @@
...
 
-Nature of CosmWasm is a perfect fit to prove again decentralized organisation and funding is efficient than old rules.
+Natirally, CosmWasm is a perfect fit to once again prove that decentralized organisation and funding is more efficient than older forms of governance.
```
Comment (orkunkl) [inline README.md:27]:
```suggestion
Naturally, CosmWasm is a perfect fit to once again prove that decentralized organisation and funding is more efficient than older forms of governance.
```

File: README.md [hunk 3]
```diff
@@ -2,68 +2,72 @@
...
 
 ## Decision Making
 
-Initially decision making will happen by communication in open/closed between foundation chats.
+Initially decision making will happen by communication in open/closed channels such as foundation chats.
+
+Each foundation will provide an address which will be used for forming a %66 percent approval multisig wallet.
 
-Each foundation will provide and address which will be used for forming %66 percent approval multisig wallet.
+Agreeing foundations will send the required funds to the project funding addresses.
```
Comment (orkunkl) [inline README.md:57]: Hmm, I thought of creating a multisig wallet, fund it then distribute to projects. 
Question how to decide amount of each foundation should send? 
Filling address first then distributing the fund seems easier.

Comment (reuvenpo) [reply README.md:57]: it looks like git got mixed up a bit. deleted line 52 here is related to added line 55

File: README.md (hunk 1 above)
Comment (reuvenpo) [reply README.md:25]: I can just remove it now

**Review by orkunkl — APPROVED** [review]

**Pull request merged**